specification/cost-management/resource-manager/readme.md

```diff
@@ -26,18 +26,30 @@ These are the global settings for the Cost Management API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-11
-azure-validator: false
+tag: package-2020-06
+azure-validator: true
```

Comment:
I remember there's an issue in modelerfour about this if azure-validator set as true. That's why the azure-cli-extension automation task fails.